alchemiscale/compute/api.py

```diff
@@ -67,17 +66,17 @@ def get_cred_compute():
 
 
 @app.get("/ping")
-async def ping():
```

Comment:
To my understanding, we don't actually want to use `async` for API endpoints that don't feature `await`s on other `async` functions. Many of our API endpoints are not IO bound, but compute bound, since there is a lot of (de)serialization happening.

If we decide later to make some of these calls take advantage of `async/await`, then we would use the `async` keyword upfront on those specifically.

Reply:
It wasn't clear to me if the perf bottleneck I was seeing on `gunicorn+uvicorn` was due to our use of `async` everywhere despite having very compute-bound API endpoints, so I went ahead and removed them.